tests/integration-tests/tests/cli_commands/test_cli_commands.py

```diff
@@ -325,6 +326,18 @@ def _test_pcluster_export_cluster_logs(s3_bucket_factory, cluster):
             bucket_cleaned_up = True
     assert_that(bucket_cleaned_up).is_true()
 
+    # test without a prefix or output file
+    ret = cluster.export_logs(bucket=bucket_name)
+    assert_that(ret).contains_key("url")
+    filename = ret["url"].split(".tar.gz")[0].split("/")[-1] + ".tar.gz"
```

Comment:
cannot be this like `filename = ret["url"].split("/")[-1]` ?

Reply:
The actual URL has a lot of additional information because it is is an s3 presigned url, so this "extra" work is to drop the signing information portion of the `url` and retrieve just the name.

Reply:
or probably we want to make sure file ends with "".tar.gz"

Reply:
See the "extra" portion after the file here:
```
$ pcluster export-cluster-logs --cluster-name ebs --bucket aws-cgruenwa-stack-export-9f27
{
  "url": "https://aws-cgruenwa-stack-export-9f27.s3.us-east-2.amazonaws.com/ebs-logs-202111230931.tar.gz?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAX2TNEWL7KZL5JE4J%2F20211123%2Fus-east-2%2Fs3%2Faws4_request&X-Amz-Date=20211123T163140Z&X-Amz-Expires=3600&X-Amz-SignedHeaders=host&X-Amz-Signature=10f1398febb827d224c979c6535966362eea41439da882d83c7811cb18cd87fc"
}
```

Reply:
oh I see, thanks